ddsrecorder_yaml/include/ddsrecorder_yaml/replayer/YamlReaderConfiguration.hpp

```diff
@@ -72,6 +73,7 @@ class DDSRECORDER_YAML_DllAPI ReplayerConfiguration
     // Specs
     unsigned int n_threads = 12;
     ddspipe::core::types::TopicQoS topic_qos{};
+    utils::LogConfiguration log_configuration{};
```

Comment:
I believe it would make more sense to store the LogConfiguration inside the DdsPipeConfiguration.